### PR TITLE
✨ Streaming v2 ✨ - GLA-958

### PR DESCRIPTION
**Pull request by nmorel**
## Summary by CodeRabbit

- **New Features**
	- Introduced new documentation entries for various API endpoints including "Download audio file," "Delete transcription," and "List transcriptions."
	- Added new files for live and pre-recorded workflows, enhancing user guidance on transcription processes.
	- New file created for "Getting started" with the Gladia API.
	- Expanded documentation for the Gladia Live Speech-To-Text (STT) API features and migration guide from V1 to V2.

- **Bug Fixes**
	- Removed deprecated API documentation to streamline user experience.

- **Documentation**
	- Enhanced clarity and structure in existing documentation, including updates to links and descriptions.
	- Updated documentation for authentication and API usage, improving clarity and usability.
	- Added new guides for getting started with the Gladia API and specific features.

- **Chores**
	- Updated variable names in documentation for consistency and clarity.
	- Introduced a new "redirects" section in the configuration file to manage URL redirections.
	- Updated `package.json` with new author and license information, and modified script commands.

### CODE REVIEW

File: api-reference/introduction.mdx [hunk 1]
```diff
@@ -4,7 +4,7 @@ description: 'Technical documentation for Gladia API'
 ---
 
 <Note>
-  If you're not looking for technical API reference documentation, you can check our [Quickstart guide](/chapters/get-started/pages/transcribe-audio).
+  If you're not looking for technical API reference documentation, you can check our [Quickstart guide](/chapters/introduction/getting-started).
```
Comment (tnesztler) [inline api-reference/introduction.mdx:7]: Could have a CI step to automatically detect if an URL path is no longer valid?

File: api-reference/v2/live/callback/audio-chunk-ack.mdx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+title: Audio chunk ack
```
Comment (tnesztler) [inline api-reference/v2/live/callback/audio-chunk-ack.mdx:2]:
```suggestion
title: Audio chunk acknowledgement
```
We should refrain from using abbreviations or acronyms inside of the documentation

File: api-reference/v2/live/init.mdx [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+title: Initiate a transcription
```
Comment (tnesztler) [inline api-reference/v2/live/init.mdx:2]:
```suggestion
title: Initiate a session
```
?

File: api-reference/v2/live/message/post-transcript.mdx [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+title: Post Transcript
```
Comment (tnesztler) [inline api-reference/v2/live/message/post-transcript.mdx:2]:
```suggestion
title: Transcript
```
To be consistent with others that removed the prefix "Post"

File: chapters/audio-intelligence/pages/translation.mdx [hunk 5]
```diff
@@ -58,7 +58,7 @@ async function pollForResult(resultUrl: string, headers: any) {
 }
 
 async function startTranscription() {
-  const gladiaKey = "YOUR_GLADIA_API_TOKEN";
+  const gladiaKey = "YOUR_GLADIA_API_KEY";
```
Comment (tnesztler) [inline chapters/audio-intelligence/pages/translation.mdx:61]: Could we also update the headers to be `X-Gladia-Key` to match proper capitalization of `X-` custom headers?

File: chapters/guides/integration.mdx [hunk 6]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Integration guides
+title: Integration
 description: "Some code samples on how to integrate Gladia to third party services"
```
Comment (tnesztler) [inline chapters/guides/integration.mdx:3]: Missing `.` at the end - same as all other descriptions

File: chapters/introduction/getting-started.mdx [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+- Speaker recognition (diarization)
+- Audio intelligence models (Translation, Summarization, Chapterization, Custom AI process...)
+- Word level timestamps
+- Subtitles generation
+- Custom vocabulary & Context prompting
+
+
+Here are some links that can help you dive deeper into our powerful API:
+
+- [Pre-recorded STT features](/chapters/pre-recorded-stt/features)
```
Comment (tnesztler) [inline chapters/introduction/getting-started.mdx:74]:
```suggestion
- [Pre-recorded Speech To Text (STT) features](/chapters/pre-recorded-stt/features)
```

File: chapters/introduction/getting-started.mdx [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+      <img src='/assets/images/playground-3.png'/>
+    </Frame>
+  </Step>
+</Steps>
+
+
+## Gladia's APIs 
+
+- [Pre-recorded Speech To Text](/chapters/pre-recorded-stt/getting-started)
+- [Live Speech To Text](/chapters/live-stt/getting-started)
```
Comment (tnesztler) [inline chapters/introduction/getting-started.mdx:54]:
```suggestion
- [Pre-recorded Speech To Text (STT)](/chapters/pre-recorded-stt/getting-started)
- [Live Speech To Text (STT)](/chapters/live-stt/getting-started)
```

File: chapters/introduction/introduction.mdx [hunk 9]
```diff
@@ -13,23 +13,23 @@ in both live and asynchronous ways, with audio intelligence tools to extract, an
...
-    href="/chapters/get-started/pages/configure-account"
+    href="/chapters/introduction/getting-started#setup-your-account"
   >
     Setup your account quickly and get your API key.
   </Card>
   <Card
     title="Transcribe an audio file"
     icon="microphone-lines"
-    href="/chapters/get-started/pages/transcribe-audio"
+    href="/chapters/introduction/getting-started#gladias-playground"
```
Comment (tnesztler) [inline chapters/introduction/introduction.mdx:25]:
```suggestion
    href="/chapters/introduction/getting-started#gladia-playground"
```
Should we remove the `s` here and in matching anchors?

File: chapters/limits-and-specifications/languages.mdx [hunk 10]
```diff
@@ -1,19 +1,15 @@
...
-Below, you'll find the list of supported languages.
-
 
 <Note>
-If you don't know which language you have please use the [Automatic Single Language Detection configuration](https://docs.gladia.io/chapters/speech-to-text-api/pages/speech-recognition#language-behaviour)
+If you don't know which language you have please use the [Automatic Single Language Detection configuration](/chapters/pre-recorded-stt/features#automatic-language-detection).
 </Note>
 
-please us the ISO 2-Letter Code in the configuration.
+In our API configuration and results, languages are defined with a 2-Letter code based on the ISO 639-1 standard.
```
Comment (tnesztler) [inline chapters/limits-and-specifications/languages.mdx:10]: We have 3 letter codes as well no?

File: package.json [hunk 11]
```diff
@@ -3,15 +3,16 @@
   "version": "1.0.0",
   "description": "Gladia documentation",
   "main": "index.js",
+  "author": "",
+  "license": "ISC",
   "scripts": {
+    "mintlify": "mintlify",
```
Comment (tnesztler) [inline package.json:9]: Was that necessary?

File: package.json [hunk 12]
```diff
@@ -3,15 +3,16 @@
   "version": "1.0.0",
   "description": "Gladia documentation",
   "main": "index.js",
+  "author": "",
```
Comment (tnesztler) [inline package.json:6]: Should we specify the author as being one of our shared email address or just "Gladia Tech team"?

File: snippets/live-flow.mdx [hunk 13]
```diff
@@ -0,0 +1,192 @@
...
+It's important to correctly define the properties `encoding`, `sample_rate`, `bit_depth` and `channels` as we 
+cannot guess them and we use those to parse your audio chunks.
+
+<CodeGroup>
+```javascript JavaScript
+  const response = await fetch('https://api.gladia.io/v2/live', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      'X-GLADIA-KEY': '<YOUR_GLADIA_API_KEY>',
```
Comment (tnesztler) [inline snippets/live-flow.mdx:15]:
```suggestion
      'X-Gladia-Key': '<YOUR_GLADIA_API_KEY>',
```

File: api-reference/introduction.mdx (hunk 1 above)
Comment (sboudouk) [reply api-reference/introduction.mdx:7]: `mintlify broken-links`

Comment (sboudouk) [reply api-reference/introduction.mdx:7]: Confusing, but it's being ran under `npm test` ci step

image

File: api-reference/v2/live/message/post-transcript.mdx (hunk 4 above)
Comment (sboudouk) [reply api-reference/v2/live/message/post-transcript.mdx:2]: I think this one refers to post transcript event, not `POST transcript`

File: package.json (hunk 11 above)
Comment (sboudouk) [reply package.json:9]: I suppose to avoid conflict with `-g mintlify`

File: chapters/introduction/introduction.mdx (hunk 9 above)
Comment (sboudouk) [reply chapters/introduction/introduction.mdx:25]: Mintlify is smart enough to handle this

**Review by sboudouk — APPROVED** [review]

**Pull request merged**